lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -78,6 +82,20 @@ module.exports = {
     };
   },
 
+  validateConditions(conditions, functionName) {
+    // At least one condition must be set
+    // https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listenerrule.html#cfn-elasticloadbalancingv2-listenerrule-conditions
+
+    if (Object.keys(conditions).length > 0) {
+      return;
+    }
+
+    throw new this.serverless.classes.Error(
+      `Cannot setup "alb" event in "${functionName}" function: At least one condition must be defined`,
+      'VALIDATION_FAILURE'
```

Comment:
Code needs to be specific and describe specific error. I propose to have it as `ALB_NO_CONDITIONS`

Reply:
Updated :)